less/admin/PermissionsPage.less

```diff
@@ -1,15 +1,9 @@
 .PermissionsPage-groups {
   background: @control-bg;
   border-radius: @border-radius;
-  max-width: calc(~'100% - 60px');
   display: block;
-  margin-left: 30px;
   overflow-x: auto;
   padding: 10px 0 8px;
-
-  > .container {
-    padding: 0 10px;
-  }
```

Comment:
sorry, another thing I noticed is since we removed the container here we need to fix the parent element's paddings, `padding: 10px;` would work just fine